src/main/java/seedu/address/model/Model.java

```diff
@@ -38,54 +36,13 @@ public interface Model {
      */
     void setGuiSettings(GuiSettings guiSettings);
 
-    /**
-     * Replaces address book data with the data in {@code addressBook}.
-     */
-    void setAddressBook(ReadOnlyAddressBook addressBook);
-
-    /** Returns the AddressBook */
-    ReadOnlyAddressBook getAddressBook();
-
-    /**
-     * Returns true if a person with the same identity as {@code person} exists in the address book.
-     */
-    boolean hasPerson(Person person);
-
-    /**
-     * Deletes the given person.
-     * The person must exist in the address book.
-     */
-    void deletePerson(Person target);
-
-    /**
-     * Adds the given person.
-     * {@code person} must not already exist in the address book.
-     */
-    void addPerson(Person person);
-
-    /**
-     * Replaces the given person {@code target} with {@code editedPerson}.
-     * {@code target} must exist in the address book.
-     * The person identity of {@code editedPerson} must not be the same as another existing person in the address book.
-     */
-    void setPerson(Person target, Person editedPerson);
-
     /**
      * Replaces the given transaction {@code target} with {@code editedTransaction}.
      * {@code target} must exist in the transaction list.
      * The transaction identity of {@code editedTransaction} must not be the same as another existing transaction.
```

Comment:
not really relevant to this PR review but just to clarify, we are defining transaction "identity" with Transaction.equals() right? does this mean that for e.g. a user won't be able to buy the same item in the same shop within the same hour? do we wanna (later on) consider implementing UID() instead?

Reply:
If i'm not wrong, we are allowing for duplicate transactions, meaning that they will be allowed to input the same item in the same shop immediately after even if it's the "same". Transaction identity should be by index for now, but we can consider implementing UID() later on for sure